Content.Server/MassMedia/Systems/NewsSystem.cs

```diff
@@ -324,4 +346,51 @@ private void OnRequestArticleDraftMessage(Entity<NewsWriterComponent> ent, ref N
     {
         UpdateWriterUi(ent);
     }
+
+    #region Discord Hook
+
+    private void OnRoundEndMessageEvent(RoundEndMessageEvent ev)
+    {
+        var query = EntityManager.EntityQueryEnumerator<StationNewsComponent>();
+
+        while (query.MoveNext(out _, out var comp))
+        {
+            foreach (var article in comp.Articles)
+            {
+                SendDiscordWebhook(article);
+            }
+        }
+    }
+
+    private async void SendDiscordWebhook(NewsArticle article)
+    {
+        if (_webhookId is null)
+            return;
+
+        try
+        {
+            var embed = new WebhookEmbed
+            {
+                Title = article.Title,
+                Description = article.Content,
```

Comment:
this should probably take a trip through FormattedMessage to plaintext it, or if you're feeling fancy, FormattedMessage -> markdown it